libs/core/langchain_core/output_parsers/list.py

```diff
@@ -6,7 +6,6 @@
 from collections import deque
 from collections.abc import AsyncIterator, Iterator
 from io import StringIO
-from typing import Optional as Optional
```

Comment:
These were intentionally due to pydantic forward references not being properly resolved. Do you know if the underlying issue is resolved? We'd potentially be seeing issues when running langchain or langchain community unit tests

Reply:
The tests pass, so I guess it's OK ?
Do you have the pydantic reference issue ?

Reply:
got it -- if our community tests are passing as well, probably OK